app/adapters/storage-addon-provider.ts

```diff
@@ -0,0 +1,13 @@
+import AddonServiceAdapter from './addon-service';
+
+export default class StorageAddonProviderAdapter extends AddonServiceAdapter {
+    pathForType() {
+        return 'storage_providers';
```

Comment:
Should we have the endpoint changed to `storage_addon_providers` in the API contract so we don't have to do this? It probably doesn't matter that much, but it might be nice to be consistent.

Reply:
That's a good point. I will ask in the API Contract doc.

Reply:
No answer yet. We can leave for now and update when they respond.